pyproject.toml

```diff
@@ -248,6 +248,8 @@ module = [
     "bokeh.ext.*",
     "bokeh.io.*",
 
+    "bokeh.layouts.*",
```

Comment:
This should be `tests.unit.bokeh.test_layouts__typing` and moved to the bottom. Ideally it would be something like `tests.unit.**.*__typing`, but I'm not sure about the syntax (it's not important given only one file for now).